src/app/public/modules/sort/sort-item.component.ts

```diff
@@ -31,9 +31,16 @@ let sortItemIdNumber: number = 0;
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SkySortItemComponent implements OnInit, OnChanges, OnDestroy {
+
+  /**
+   * Indicates whether the sorting option is active.
+   */
   @Input()
   public active: boolean;
 
+  /**
+   * Emitted when a sort item is clicked.
```

Comment:
`clicked` should be `selected`. Also, this doesn't follow our patterns on event descriptions. Can we change this to `Fires when a sort item is selected.`? (I'd prefer `Fires when users select a sort item.`, but that doesn't account for the possibility that the button is selected programmatically.)

Reply:
Done.